presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

```diff
@@ -1132,6 +1132,66 @@ TEST_F(TaskManagerTest, checkBatchSplits) {
   ASSERT_EQ(resultOrFailure.status, nullptr);
 }
 
+TEST_F(TaskManagerTest, spillDirectoryNonExist) {
+  // Cleanup old tasks between test iterations.
+  taskManager_->setOldTaskCleanUpMs(0);
+  for (bool spillDirExist : {false, true}) {
+    SCOPED_TRACE(fmt::format("spillDirExist: {}", spillDirExist));
+    auto spillDir = setupSpillPath();
+
+    const int numBatchesPerFile = 5;
```

Comment:
Do we need 5 batches of 1000 rows each and 5 files? Can we make 1 batch with 10 rows and use Values node instead of TableScan? This will make this test run faster.

Reply:
Not required